stdlib/errno.pyi

```diff
@@ -91,9 +91,15 @@ ECANCELED: int  # undocumented
 ENOTRECOVERABLE: int  # undocumented
 EOWNERDEAD: int  # undocumented
 
+if sys.platform == "sunos5" or sys.platform == "solaris":
```

Comment:
flake8-pyi is just flagging this because it doesn't know about these platforms. I _think_ they should work fine with type checkers, so we should just `noqa` them?

```suggestion
if sys.platform == "sunos5" or sys.platform == "solaris":  # noqa: Y008
```

Reply:
We can also add these platforms to the list of supported platforms in flake8-pyi. I think I came up with that list years ago purely based on what was in use in typeshed at the time.

Reply:
I'll noqa them for now. If we add them to flake8-pyi, we should double check whether "solaris" is actually a valid platform. It's only used in CPython source code, in a tools, while "sunos5" is used multiple times.